docs/feature_flags.md

```diff
@@ -0,0 +1,11 @@
+# Feature Flags
+
+This demo comes with 2 feature flags which control failure conditions in the
+Product Catalog and Shipping services. By default the flags are disabled. Using
```

Comment:
This first sentence would need to change with the addition of more feature flags so I think we can remove it